webapp/channels/src/actions/views/root.ts

```diff
@@ -107,3 +155,76 @@ export function registerCustomPostRenderer(type: string, component: any, id: str
         return {data: true};
     };
 }
+
+export function redirectToOnboardingOrDefaultTeam(history: History): ThunkActionFunc<void> {
```

Comment:
brought in from components/root/actions

Reply:
I've actually been preferring to move component-specific actions like these closer to the components themselves rather than having them in `actions/views`. I don't expect we're likely to reuse methods like this, and it makes it easier to keep the logic self-contained if it lives next to the corresponding component

Reply:
ok will move all of these back to component's actions file